docs/usecases/reproducible_neuroimaging_analysis.rst

```diff
@@ -0,0 +1,286 @@
+
+For neuroscientific studies, :term:`the DataLad superdataset ///` provides unified
+access to a large amount of data. Using it to install datasets into an
+analysis-superdataset makes it easy to share this data together with the analysis.
+By ensuring that all relevant data is downloaded via :command:`datalad get`
+via DataLad's command line tools in the analysis scripts, or ``--input`` specifications
+in a :command:`datalad run`, an analysis can retrieve all required
+inputs fully automatically during execution.
+Recording executed commands with :command:`datalad run` allows to rerun
+complete analysis workflows with a single command, even if input data does not exist
+locally. Combining these three steps allows to share fully automatically reproducible
+analyses as lightweight datasets.
+
+Step-by-Step
+^^^^^^^^^^^^
+
+It always starts with a dataset:
+
+.. runrecord:: _examples/repro-101
+   :language: console
+   :workdir: usecases/repro
+
+   $ datalad create -c yoda demo
+
+For this demo we are using two public brain imaging datasets that were published on
+`OpenFMRI.org <https://legacy.openfmri.org/>`_, and are available from
+:term:`the DataLad superdataset ///` (datasets.datalad.org). When installing datasets
+from this superdataset, we can use its abbreviation ``///``.
+The two datasets, `ds000001 <https://legacy.openfmri.org/dataset/ds000001/>`_ and
+`ds000002 <https://legacy.openfmri.org/dataset/ds000002/>`_, are installed into the
+subdirectory ``inputs/``.
+
+.. runrecord:: _examples/repro-102
+   :language: console
+   :workdir: usecases/repro
+
+   $ cd demo
+   $ datalad install -d . -s ///openfmri/ds000001 inputs/ds000001
+
+.. runrecord:: _examples/repro-103
+   :language: console
+   :workdir: usecases/repro
+
+   $ cd demo
+   $ datalad install -d . -s ///openfmri/ds000002 inputs/ds000002
+
+Both datasets are now registered as subdatasets, and their precise versions
+(e.g. in the form of the commit shasum of the lastest commit) are on record:
+
+.. runrecord:: _examples/repro-104
+   :language: console
+   :workdir: usecases/repro/demo
+
+   $ datalad --output-format '{path}: {gitshasum}' subdatasets
+
+DataLad datasets are fairly lightweight in size, they only contain
+pointers to data and history information in their minimal form.
+Thus, so far very little data were actually downloaded:
+
+.. runrecord:: _examples/repro-105
+   :language: console
+   :workdir: usecases/repro/demo
+
+   $ du -sh inputs/
+
+Both datasets would actually be several gigabytes in size, once the
+dataset content gets downloaded:
+
+.. runrecord:: _examples/repro-106
+   :language: console
+   :workdir: usecases/repro/demo
+
+   $ datalad -C inputs/ds000001 status --annex
+   $ datalad -C inputs/ds000002 status --annex
+
+Both datasets contain brain imaging data, and are compliant with the
+`BIDS standard <https://bids.neuroimaging.io/>`_.
+This makes it really easy to locate particular images
+and perform analysis across datasets.
```

Comment:
Mental note: produce a different use case that actually find images based on their metadata, not on filename conventions. Or actually, enhance this usecase to do that, to better illustrate the potential of annotated public data.
Possibly cross-reference with https://github.com/datalad-handbook/book/pull/202

Reply:
I like this idea a lot. I made an issue about this here: https://github.com/datalad-handbook/book/issues/209